src/backend/pluginLoaders.js

```diff
@@ -97,13 +101,41 @@ export async function updateMirror(
   return {project, cache};
 }
 
+/**
+ * Creates a ReferenceDetector composing all plugin reference detectors
+ * requested by the project.
+ */
+export async function createReferenceDetector(
+  {github, discourse}: $Shape<PluginLoaders>,
+  {githubToken}: GraphEnv,
+  {cache, project}: CachedProject
+): Promise<ReferenceDetector> {
+  const refs = [];
```

Comment:
Nit: would be nice to have an explicit typesig on refs

Reply:
Done!